src/scenes/Boot.ts

```diff
@@ -28,4 +54,30 @@ export class Boot extends TenebrisScene {
       );
     });
   }
+
+  update() {
+    const forwardKey = this.input.keyboard?.addKey(
```

Comment:
can we put all controls in a seperate file to develop custom controls in the future?

Reply:
yup, that sounds more organised